index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>My portfolio</title>
+</head>
+<body>
+    
+    <header>
+        <ul class="top mobile">
+            <li> <a href="#">9:15</a> </li>
+            <li> <a href="#"><img src="images/wifi.png" alt=""></a></li>        
+        </ul>
```

Comment:
- [ ] [OPTIONAL] Kindly remove this section as it is not part of the design.

Reply:
- [ ] [Optional] This change has not been implemented.